### DISCUSSION TITLE
Release Status 22.0.8 Keycloak version

### DISCUSSION THREAD

**Discussion by Gokula-1** [keycloak/keycloak#27165] [Q&A] [answered]
Hi all,
I wanted to enquire about the Keycloak 22.0.8 version in the repository. There is no details about this particular version in any of the Keycloak release documentation. The Keycloak website's download section doesn't contain any package with the 22.0.8 tag as well.

So, 
I just wanted to enquire what is the official status of the 22.0.8 version of Keycloak. Can it be taken as a GA version or is it just a release candidate 
Thank you for your replies

**Comment by vmuzikar** [comment, marked as answer]
Keycloak 22 is no longer supported here in the upstream and no further releases are planned (including 22.0.8 version). We strongly suggest upgrading to the latest version – currently Keycloak 23.

If you are interested in a fully supported LTS version, checkout e.g. [Red Hat build of Keycloak](https://access.redhat.com/products/red-hat-build-of-keycloak) which offers the 22.0.8 version you are asking about. That is a GA version, not a release candidate.
